eos-tech-support/eos-hack-reset-first-contact

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+flag_file=/var/eos-hack-reset-first-contact
+
+# Check that script was run with superuser privileges.
+if [[ $EUID != 0 ]]; then
+    echo "$0 requires superuser privileges."
+    exit 1
+fi
+
+# Check if flag file exists
+if [ ! -f $flag_file ]; then
+    echo "$0: flag file '$flag_file' does not exist"
+    exit 1
+fi
+
+rm $flag_file
+
+IFS=$'\n'
+for user_row in $(cat /etc/passwd); do
+    user=$(echo $user_row | cut -d : -f 1)
+    user_id=$(echo $user_row | cut -d : -f 3)
+
+    if [[ $user_id -ge 1000 && $user_id -le 1999 ]]; then
+        delete_tries=1
```

Comment:
What is the thinking behind the retry loop? Under what circumstances are you expecting userdel to fail, and later succeed?

Reply:
When you isolate `eos-hack-reset-first-contact.target`, the processes used by the users take some time to finish.

Reply:
Could you achieve similar with After=systemd-kill-everything? (Whatever it's called.)